### DISCUSSION TITLE
(필독) Computer-Science-Collections 커뮤니티에 오신 여러분을 환영합니다!

### DISCUSSION THREAD

**Discussion by Irisation23** [cs-collections/cs-collections-discussion#3] [a. 공지사항]
`Computer-Science-Collections` 커뮤니티는 참여하신 여러분들과 함께 학습하고 성장할 수 있는 공간입니다. 
`Computer-Science-Collections`에는 여러 학습인원들과 함께 할 수 있는 여러 활동이 있습니다. 
자유로운 학습 컨텐츠 공유를 위해 집단 지성의 힘을 발휘해 볼 수 있는 커뮤니티가 될 수 있도록 많은 관심과 참여 부탁드립니다. 

감사합니다. 🙏🏻

> 해당 커뮤니티의 규칙과 룰은 깃허브가 제공하는 기본 룰을 따르고 있습니다.
> 자세한 내용은 [공지사항](https://github.com/orgs/cs-collections/discussions/1)을 참조 해주세요.

**Comment by Irisation23** [comment]
저는 Back End Developer로 성장하고 싶은 @bunsung92 라고 합니다.
저와 `Contributor` @JoisFe 는 깊이 있는 **CS** 학습을 위해 해당 Discussion을 이용하려고 합니다.
부족한 점이 많을 수 있습니다.
많이 배우며 성장할 수 있도록 도와주시면 감사하겠습니다. 🙇🏻‍♂️

**Reply by JoisFe** [reply]
같이 열심해 해봐요^^

**Comment by JoisFe** [comment]
서비스 개발에 대한 관심과 흥미를 넘어 서비스를 사용할 클라이언트들에게 편의와 도움을 줄 수 있는 백엔드 개발자로 성장하고 싶은 @JoisFe 입니다.
혼자 CS를 공부하는 것 보단 다양한 사람들과 함께 의견을  공유해가며 학습할 수 있다면 좋겠다는 생각이 컸는데
해당 디스커션을 통해서 다 같이 성장할 수 있으면 좋겠습니다. 🐢
